src/components/leftDrawer.js

```diff
@@ -0,0 +1,301 @@
+import React from 'react';
+import { Link as RouterLink } from 'react-router-dom';
+import Drawer from '@material-ui/core/Drawer';
+import IconButton from '@material-ui/core/IconButton';
+import ChevronLeftIcon from '@material-ui/icons/ChevronLeft';
+import Divider from '@material-ui/core/Divider';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ForumIcon from '@material-ui/icons/Forum';
+import MenuBookIcon from '@material-ui/icons/MenuBook';
+import EqualizerIcon from '@material-ui/icons/Equalizer';
+import AccountBalanceWalletIcon from '@material-ui/icons/AccountBalanceWallet';
+import SettingsIcon from '@material-ui/icons/Settings';
+import '../styles/leftDrawer.css'
+
+function forumRouterStyle(location) {
+  if (location === "forum") {
+    return {
+      color: "#536dfe"
+    };
+  }
+  else {
+    return {
+      color: "#757575"
+    };
+  }
```

Comment:
Consider using ternary operator. Can be made shorter with this statement:
```javascript
function forumRouterStyle(location) {
  return {
    color: location === "forum" ? "#536dfe" : "#757575",
  };
}
```

Would be better if we use our color palette from Material-UI for consistency. Not sure how. New PR if this takes too long.